.travis.yml

```diff
@@ -87,3 +87,18 @@ jobs:
         - if [ "${TRAVIS_PULL_REQUEST}" != "false" ]; then
             bash -x tools/lint-pr-commit-message.sh ${TRAVIS_PULL_REQUEST};
           fi
+
+    - name: "Python 3 is EXPERIMENTAL"
+      language: node_js
+      node_js: "node"
+      install:
+        - pyenv global 3.7.1
+        - python3.7 -m pip install --upgrade pip
+        - make lint-py-build
+      script:
+        - NODE=$(which node) make lint lint-py
+        - python3.7 ./configure.py
+        - NODE=$(which node) make test
```

Comment:
You could also prefix this with `PYTHON=python3.7` to run tools/gyp_node.py and tools/test.py with Python 3 too. There's also an `install` target that could exercise tools/install.py, but maybe that can be added later. It'll be important for making sure that release builds can be done with Python 3.

Reply:
Thanks.  Will add these after `make test` goes green.